commands/cli.go

```diff
@@ -41,14 +46,26 @@ func awsNuke(c *cli.Context) error {
 	for _, excludedRegion := range excludedRegions {
 		if !collections.ListContainsElement(regions, excludedRegion) {
 			return InvalidFlagError{
-				Name: "exclude-regions",
+				Name:  "exclude-regions",
 				Value: excludedRegion,
 			}
 		}
 	}
 
+	var excludeSince time.Time
+	var err error
+
+	if c.String("exclude-since") != "" {
+		excludeSince, err = time.Parse("01-02-2006 03:04AM", c.String("exclude-since"))
+		if err != nil {
+			return errors.WithStackTrace(err)
+		}
+	} else {
+		excludeSince = time.Now()
+	}
```

Comment:
I'd extract the logic above into its own method with a signature like `parseTime(paramName string, c *cli.Context) (time.Time, error)`. It would be more readable, reusable, testable, and you could then simply return the parsed value or an error without any mutable variables.